src/oneD/refine.cpp

```diff
@@ -12,7 +12,7 @@ namespace Cantera
 {
 Refiner::Refiner(Domain1D& domain) :
     m_ratio(10.0), m_slope(0.8), m_curve(0.8), m_prune(-0.001),
-    m_min_range(0.01), m_domain(&domain), m_npmax(3000),
+    m_min_range(0.01), m_domain(&domain), m_npmax(300),
```

Comment:
You're right, I think I was just annoyed that I was waiting for simulations to finish that took forever because they never converged and hit the 3000 limit, so I was thinking a "fail early" limit would be better. Would 500 or 1000 be more reasonable?